
syntax = "proto3";

package security;

option java_multiple_files = true;
option java_package = "common.model.protos";
option java_outer_classname = "SecurityQuantityTypeProtos";

enum SecurityQuantityTypeProto {

  UNKNOWN_QUANTITY_TYPE = 0; //Maps to Security
  ORIGINAL_FACE_VALUE = 1;
  NOTIONAL = 2;
  UNITS = 3;
}